launch/interaction/vizbox.launch

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+
```

Comment:
This looks really messy in terms of spacing ;-)

Reply:
@LoyVanBeek